tests/app/questionnaire/test_return_location.py

```diff
@@ -0,0 +1,37 @@
+from app.questionnaire.return_location import ReturnLocation
+
+
+def test_return_location_to_dict():
+    return_location = ReturnLocation(
```

Comment:
Minor but could reuse this object as it is the same for both tests? Or parametrize?